control-plane/subcommand/inject-connect/command.go

```diff
@@ -537,6 +537,7 @@ func (c *Command) Run(args []string) int {
 			ConsulTLSServerName:        c.consul.TLSServerName,
 			ConsulPartition:            c.consul.Partition,
 			ConsulCACert:               string(caCertPem),
+			InitContainerResources:     &initResources,
```

Comment:
This is the only difference from the source PR that is being backported. The structure of this file changed drastically between 1.2.x and 1.3.x